packages/ai/src/utils/tw-to-ws.ts

```diff
@@ -0,0 +1,32 @@
+import type { WsEmbedTemplate } from "@webstudio-is/react-sdk";
+import { traverseTemplateAsync } from "./traverse-template";
+import { parseTailwindToWebstudio } from "@webstudio-is/css-data";
+
+export const tailwindToWebstudio = async (template: WsEmbedTemplate) => {
+  return traverseTemplateAsync(template, async (node, parent) => {
+    if (node.type === "instance" && node.props && node.props.length > 0) {
+      const classNameProp = node.props.find(
+        (prop) => prop.name === "className"
+      );
+      if (classNameProp && classNameProp.type === "string") {
+        const styles = await parseTailwindToWebstudio(classNameProp.value);
+
+        if (Array.isArray(node.styles)) {
```

Comment:
Why is array check? Isn't it instance node? type is always array or undefined.

Reply:
we usually check for undefined so we know what's being checked explicitly in this case